packages/jellyfish-api-core/__tests__/category/account/accountToUtxos.test.ts

```diff
@@ -0,0 +1,98 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BalanceTransferPayload, UTXO } from '../../../src/category/account'
+import { RpcApiError } from '../../../src'
+
+describe('Account with DBTC', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let from: string
+
+  async function setup (): Promise<void> {
+    from = await container.call('getnewaddress')
+    await createToken(from, 'DBTC')
+  }
+
+  async function createToken (address: string, symbol: string): Promise<void> {
+    const metadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+    await container.waitForWalletBalanceGTE(101)
+    await container.call('createtoken', [metadata])
+    await container.generate(1)
+
+    await container.call('utxostoaccount', [{ [address]: '100@0' }])
+    await container.generate(1)
+  }
+
+  it('should accountToUtxos', async () => {
+    const balanceBefore = await container.call('getbalance')
+
+    const payload: BalanceTransferPayload = {}
+    // NOTE(jingyi2811): Only support sending from account to DFI Utxos.
+    payload[await container.getNewAddress()] = '5@DFI'
+    payload[await container.getNewAddress()] = '5@DFI'
+
+    const data = await client.account.accountToUtxos(from, payload)
+
+    expect(typeof data).toStrictEqual('string')
+    expect(data.length).toStrictEqual(64)
+
+    const balanceAfter = await container.call('getbalance')
+    expect(balanceAfter).toBeGreaterThan(balanceBefore)
```

Comment:
should assert the `after = before + new`

Reply:
<img width="180" alt="Screenshot 2021-06-01 at 1 01 43 PM" src="https://user-images.githubusercontent.com/7101806/120269333-2f783c00-c2da-11eb-9d89-dca3871720ad.png">

Unable to get 10. 105.6647572 - 95.6648398 = 9.9999174. I guess there is a fee.

Reply:
Yeah, it's too hard to test this because the balance might get updated with new blocks minted.

Reply:
However, you can check your account balance. That will always be consistent.